src/modules/core/components/Fields/InputLabel/InputLabel.tsx

```diff
@@ -73,7 +75,13 @@ const InputLabel = ({
       htmlFor={inputId || null}
     >
       <span className={styles.labelText}>{labelText}</span>
-      {helpText && <span className={styles.help}>({helpText})</span>}
+      {helpText && (
+        <span className={styles.help}>
+          <span className={styles.paren}>(</span>
```

Comment:
What's up with these hidden parens? Do we need them?

Reply:
My thought was, to provide context (that this **is in fact** help text) to screen reader users, we should keep the parens in the DOM, while visually hiding them for sighted users (since the right alignment provides the context for this group of users). The parens will still be displayed if the help text is left aligned.

☝️ that said, I'm realizing that screen readers won't read the parens with `display: none`, so I'll need to adjust them to be hidden via `text-indent` or another method.

Reply:
I made all the parens parts of the messages, which I think is a bit safer for purposes of i18n.